service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1466,12 +1465,25 @@ components:
 
     RelationshipFilterV2:
       type: object
-      description: Filter on a relationship between entities (e.g. condition occurrences where condition=diabetes)
+      description: |
+        Filter on a relationship between entities (e.g. condition occurrences where condition=diabetes).
+        If one of the group_by fields are set, they must all be set. If group_by_fields are set:
+        - First subfilter runs
+        - subfilter results are grouped by group_by_count_attribute
+        - Instance will be returned only if there are group_by_count_operator group_by_count_value results for that
+          instance
+        group_by fields can be used to filter on # of occurrence dates.
       properties:
         entity:
           type: string
         subfilter:
           $ref: "#/components/schemas/FilterV2"
+        group_by_count_attribute:
+          type: string
+        group_by_count_operator:
+          $ref: "#/components/schemas/BinaryOperatorV2"
+        group_by_count_value:
+          $ref: "#/components/schemas/LiteralV2"
```

Comment:
This should be an integer rather than a literal, since it's being compared to the count.

Reply:
Done, thanks.